tests/javascript-config/unit/fixtures/datetime-response-authed.js

```diff
@@ -0,0 +1,63 @@
+export const AuthedDateTimeResponse = {
```

Comment:
So the fixtures folder in here is a place for us to keep any general fixtures that we might use across various tests.  Jest is configured to expose these fixtures on `@test/fixtures` so you can import from there anywhere you need a fixture.